From 9749a34422340ee41c22c9d1f90051430a8d1768 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 19 Jun 2015 08:54:56 +0800
Subject: [PATCH] dts: rk312x-pinctrl: disable IO pull-down for SDIO

  Rockchips' platform contain ridiculous and closely
w/o maintained macro in pcl to pave the bumpy way for
other drivers to distinguish VALUE_PULL_UPDOWN_DISABLE
from VALUE_PULL_DISABLE.

Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
Signed-off-by: Xiao Yao <xiaoyao@rock-chips.com>
Tested-by: Xiao Yao <xiaoyao@rock-chips.com>
Cc: Roger Hu <roger.hu@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-pinctrl.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/rk312x-pinctrl.dtsi b/arch/arm/boot/dts/rk312x-pinctrl.dtsi
index bea2fe80cedc..197fe6640d34 100755
--- a/arch/arm/boot/dts/rk312x-pinctrl.dtsi
+++ b/arch/arm/boot/dts/rk312x-pinctrl.dtsi
@@ -662,22 +662,22 @@
 
 			sdio0_cmd: sdio0_cmd {
 				rockchip,pins = <MMC1_CMD>;
-				rockchip,pull = <VALUE_PULL_DISABLE>;
+				rockchip,pull = <VALUE_PULL_UPDOWN_DISABLE>;
 			};
 			sdio0_clk: sdio0_clk {
 				rockchip,pins = <SDMMC_CLKOUT>;
-				rockchip,pull = <VALUE_PULL_DISABLE>;
+				rockchip,pull = <VALUE_PULL_UPDOWN_DISABLE>;
 			};
 			sdio0_bus1: sdio0-bus-width1 {
 				rockchip,pins = <SDMMC_DATA0>;
-				rockchip,pull = <VALUE_PULL_DISABLE>;
+				rockchip,pull = <VALUE_PULL_UPDOWN_DISABLE>;
 			};
 			sdio0_bus4: sdio0-bus-width4 {
 				rockchip,pins = <SDMMC_DATA0>,
 						<SDMMC_DATA1>,
 						<SDMMC_DATA2>,
 						<SDMMC_DATA3>;
-				rockchip,pull = <VALUE_PULL_DISABLE>;
+				rockchip,pull = <VALUE_PULL_UPDOWN_DISABLE>;
 			};
 
 			sdio0_gpio: sdio0_gpio{
@@ -688,7 +688,7 @@
 						<GPIO1_A2>, //data1
 						<GPIO1_A4>, //data2
 						<GPIO1_A5>; //data3
-				rockchip,pull = <VALUE_PULL_DISABLE>;
+				rockchip,pull = <VALUE_PULL_UPDOWN_DISABLE>;
 			};
 
 		};
-- 
2.35.3

